wiki/Client/Beatmap_editor/Compose/en.md

```diff
@@ -1,351 +1,257 @@
-| Lock Notes (`L`) | ("Select" on a Hit note) **Left-Click:** Lock this note | Lock selected note to current position and timestamp. |
+| `Grid Snap` (`T`) | **Hold** `Shift`: Temporary toggle. **Hold** `Ctrl`: Temporary disable angle snapping. | Notes will be snapped to the grid by default. |
+| `Distance Snap` (`Y`) | **Hold** `Alt`: Temporary toggle. Switch Beat Snap Divisor to Distance Snap. `Alt` **+** `Mousewheel`: Adjust Distance Snap multiplier. | Distance between consecutive notes will be snapped based on their rhythmical difference. Best use while the timeline is paused. |
+| `Lock Notes` (`L`) | **(Select a note)** `Left click`: Lock this note. | Lock selected note to current position and timestamp. |
 
-For **keysound** (adding SE on hitnotes), select the note and **`Alt` + Left Click** on the selected note (a pop-up window will be shown with sound samples inside your beatmap folder). Refer to "Sampling" section at end of page.
+### Bottom (Song's timeline)
 
-### Bottom (Song Timeline)
+![](img/song-timeline.jpg "Song's timeline")
 
-![Song Timeline](/wiki/shared/BE_STL.jpg "Song Timeline")
+At **bottom-left**, it shows the **timestamp** in milliseconds and the **song duration** in percent. The percentage may be switched to "intro" or "outro" if there is storyboarding before or after the music.
 
-At **bottom-left**, it shows **timestamp in milliseconds (ms)** and **song duration %**.
+At **bottom-centre**, it shows the **timeline with markings** and the **compulsory music player buttons**. The `Test` button to the right of the timeline will save your beatmap then allows you to play test it, starting at the current timestamp.
 
-At **bottom-centre**, it shows the **timeline with markings and the compulsory music player buttons.** For "Test" button, your beatmap will be saved to test the beatmap starting on current timestamp.
+At **bottom-right**, you can adjust the **playback rate** by per quarter value.
 
 #### Bookmarking command
 
 | Shortcut | Description |
 | :-- | :-- |
-| `Ctrl` + `B` | Add bookmark at current location. |
-| `Ctrl` + `Shift` + `B` | Remove bookmark at current location. |
-| `Ctrl` + `Right` Arrow | Next bookmark. |
-| `Ctrl` + `Left` Arrow | Previous bookmark. |
-
-At **bottom-right**, you can adjust the **Playback Rate** by **per quarter value.**
+| `Ctrl` + `B` | Adds a bookmark at current location. |
+| `Ctrl` + `Shift` + `B` | Removes the bookmark at current location. |
+| `Ctrl` + `Right arrow` | Moves the current timestamp to the next bookmark. |
+| `Ctrl` + `Left arrow` | Moves the current timestamp to the previous bookmark. |
 
 ### Sampling
 
-![Sample pop-up window (Windows 8's ver.)](img/Sample_omc.jpg "Sample pop-up window (Windows 8's ver.)")
+![](img/sample-import.jpg "Sample pop-up window")
```

Comment:
It's probably better to slice this image and move the `Sample events list` window part down to its corresponding chapter.